### PR TITLE
Improve Log Functionality

### PR DESCRIPTION
**Pull request by CalvinScott**
## Team

Team Lead: @thalesfsp 
Project Lead: @CalvinScott
Deployment Review: @slajax

### Overview

In reference to https://github.com/Retsly/log/issues/65, microservices is going to receive improved log functionality to better track errors/events.

### Steps

- [x] Make sure that we use Retsly/log over any other log such as console.log/logger
- [x] Check current usage of log and confirm we are sending correct and relevant information
- [x] Improve log coverage by adding additional log messages where needed

Events to track:
- [x] any time a service is started
- [x] Improve errors to be more descriptive

### CODE REVIEW

**Comment by CalvinScott** [comment]
@RongxinZhang @thalesfsp Please review when you're free

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Axon wrapper to handle Retsly microservices",
   "main": "lib/index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "node ./lib/index.js",
```
Comment (thalesfsp) [inline package.json]: node lib/index

**Comment by thalesfsp** [comment]
:+1:

**Comment by RongxinZhang** [comment]
:v: looks good

**Comment by slajax** [comment]
Please merge this and tag it as `1.3.3` release. Put notes as updating logs and bump version in all dependencies.

**Pull request merged**